src/components/Tutor/BaseURL.js

```diff
@@ -0,0 +1,68 @@
+function isValid(val) {
+  return val !== "";
+}
```

Comment:
For filters, you might want to have a different `isValid` function for each one.